app/controllers/user_organization_memberships_controller.rb

```diff
@@ -10,4 +11,14 @@ def organization_memberships
     @memberships = UserOrganizationMembership.for_organization(params[:organization_id])
     render json: OrganizationMembershipsRepresenter.new(@memberships).to_json
   end
+
+  def destroy
+    @membership = UserOrganizationMembership.find(params[:id])
+    if current_user.id == @membership.organization.admin_id
+      @membership.destroy
+      head :no_content
```

Comment:
These lines seem like they should go into a before_filter to check authorization.